src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -1764,13 +1764,27 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         .setProcessingSpeed(1)
         .register(plugin);
 
+        new AutoEnchanter(categories.electricity, SlimefunItems.AUTO_ENCHANTER_2, RecipeType.ENHANCED_CRAFTING_TABLE,
+        new ItemStack[] {SlimefunItems.REINFORCED_PLATE, SlimefunItems.BIG_CAPACITOR, SlimefunItems.REINFORCED_PLATE, SlimefunItems.BLISTERING_INGOT_3, SlimefunItems.AUTO_ENCHANTER, SlimefunItems.BLISTERING_INGOT_3, SlimefunItems.WITHER_PROOF_OBSIDIAN, SlimefunItems.SYNTHETIC_DIAMOND, SlimefunItems.WITHER_PROOF_OBSIDIAN})
+        .setCapacity(1024)
+        .setEnergyConsumption(24)
+        .setProcessingSpeed(3)
+        .register(plugin);
+
         new AutoDisenchanter(categories.electricity, SlimefunItems.AUTO_DISENCHANTER, RecipeType.ENHANCED_CRAFTING_TABLE,
         new ItemStack[] {new ItemStack(Material.REDSTONE), new ItemStack(Material.ANVIL), new ItemStack(Material.REDSTONE), SlimefunItems.CARBONADO, SlimefunItems.AUTO_ENCHANTER, SlimefunItems.CARBONADO, SlimefunItems.WITHER_PROOF_OBSIDIAN, SlimefunItems.WITHER_PROOF_OBSIDIAN, SlimefunItems.WITHER_PROOF_OBSIDIAN})
         .setCapacity(128)
         .setEnergyConsumption(9)
         .setProcessingSpeed(1)
         .register(plugin);
 
+        new AutoDisenchanter(categories.electricity, SlimefunItems.AUTO_DISENCHANTER_2, RecipeType.ENHANCED_CRAFTING_TABLE,
+        new ItemStack[] {SlimefunItems.REINFORCED_PLATE, new ItemStack(Material.ANVIL), SlimefunItems.REINFORCED_PLATE, SlimefunItems.CARBONADO, SlimefunItems.AUTO_ENCHANTER_2, SlimefunItems.CARBONADO, SlimefunItems.WITHER_PROOF_OBSIDIAN, SlimefunItems.WITHER_PROOF_OBSIDIAN, SlimefunItems.WITHER_PROOF_OBSIDIAN})
```

Comment:
This should probably require an Auto Disenchanter in the recipe. Since it is an upgrade of that.
Then perhaps swap the carbonado for blistering ingots.

Reply:
I second this

Reply:
Changed Auto Disenchanter Recipe to:
```
Auto Disenchanter II
Speed: 3x
Power: 48J/s
Buffer: 1024J

Recipe:
| A | B | A |
| C | D | C |
| E | F | E |

A = Reinforced Plate
B = Anvil
C = Blistering Ingot (100%)
D = Auto Disenchanter
E = Wither Proof Obsidian
F = Big Capacitor```
Changed wither proof obsidian with big capacitor for the buffer size.

Added new research for tier two enchanters with 48 exp levels on ID 277.